ferret/explainers/explanation_speech/loo_speech_explainer.py

```diff
@@ -1,11 +1,15 @@
 """LOO Speech Explainer module"""
+
 import numpy as np
 from typing import Dict, List, Union, Tuple
 from pydub import AudioSegment
 from IPython.display import display
 from .explanation_speech import ExplanationSpeech
-from .utils_removal import transcribe_audio, remove_word
-from ...speechxai_utils import pydub_to_np, print_log
+from .utils_removal import remove_word, remove_word_np
+from ...speechxai_utils import pydub_to_np, FerretAudio
+from logging import getLogger
+
+logger = getLogger(__name__)
```

Comment:
We are starting to use loggers in different places: let's use a singleton pattern so we avoid having multiple ones instantiated at the same time.